com.microsoft.mrtk.input/Subsystems/SyntheticHands/HandPoses/SimulatedArticulatedHandPoses.cs

```diff
@@ -4,66 +4,66 @@
 using UnityEngine;
 
-using GestureId = Microsoft.MixedReality.Toolkit.Input.GestureTypes.GestureId;
+using HandshapeId = Microsoft.MixedReality.Toolkit.Input.HandshapeTypes.HandshapeId;
 
 namespace Microsoft.MixedReality.Toolkit.Input
 {
     /// <summary>
-    /// This stores the joint pose JSON data that defines various articulated hand gestures for input simulation.
+    /// This stores the joint pose JSON data that defines various articulated hand poses for input simulation.
     /// The JSON data that defines each joint position and orientation is stored in strings to avoid file loading/targeting during runtime
     /// </summary>
     public static class SimulatedArticulatedHandPoses
     {
-        private static readonly Dictionary<GestureId, string> GesturePoseJSONMapping = new Dictionary<GestureId, string>()
+        private static readonly Dictionary<HandshapeId, string> HandshapePoseJSONMapping = new Dictionary<HandshapeId, string>()
         {
-            { GestureId.Flat, ArticulatedHandPose_Flat},
-            { GestureId.Grab, ArticulatedHandPose_Grab},
-            { GestureId.Open, ArticulatedHandPose_Open},
-            { GestureId.OpenSteadyGrabPoint, ArticulatedHandPose_OpenSteadyGrabPoint},
-            { GestureId.Pinch, ArticulatedHandPose_Pinch},
-            { GestureId.PinchSteadyWrist, ArticulatedHandPose_PinchSteadyWrist},
-            { GestureId.Poke, ArticulatedHandPose_Poke },
-            { GestureId.ThumbsUp, ArticulatedHandPose_ThumbsUp},
-            { GestureId.Victory, ArticulatedHandPose_Victory},
-            { GestureId.TeleportStart, ArticulatedHandPose_TeleportStart},
-            { GestureId.TeleportEnd, ArticulatedHandPose_TeleportEnd}
+            { HandshapeId.Flat, ArticulatedHandPose_Flat},
+            { HandshapeId.Grab, ArticulatedHandPose_Grab},
+            { HandshapeId.Open, ArticulatedHandPose_Open},
+            { HandshapeId.OpenSteadyGrabPoint, ArticulatedHandPose_OpenSteadyGrabPoint},
+            { HandshapeId.Pinch, ArticulatedHandPose_Pinch},
+            { HandshapeId.PinchSteadyWrist, ArticulatedHandPose_PinchSteadyWrist},
+            { HandshapeId.Poke, ArticulatedHandPose_Poke },
+            { HandshapeId.ThumbsUp, ArticulatedHandPose_ThumbsUp},
+            { HandshapeId.Victory, ArticulatedHandPose_Victory},
+            { HandshapeId.TeleportStart, ArticulatedHandPose_TeleportStart},
+            { HandshapeId.TeleportEnd, ArticulatedHandPose_TeleportEnd}
         };
 
-        private static Dictionary<GestureId, HandJointPose[]> gesturePoses;
+        private static Dictionary<HandshapeId, HandJointPose[]> handshapePoses;
 
         /// <summary>
-        /// Get pose data for a supported gesture.
+        /// Get pose data for a supported handshape.
         /// </summary>
-        public static void GetGesturePose(GestureId gesture, out HandJointPose[] poseData)
+        public static void GetHandshapePose(HandshapeId handshape, out HandJointPose[] poseData)
```

Comment:
Should we drop `Pose` from all these methods as well, to match the enum and whatnot?

Reply:
or `GetHandshapePoses` if we're referring to the hand joint poses representing a handshape?

Reply:
I'd rather clarify it as `GetHandshapeJointPoseData`, since we go from the concept of "a certain handpose" to the more concrete joint data which it represents.